### DISCUSSION TITLE
Portfolio Diversity

### DISCUSSION THREAD

**Discussion by mkitzmann** [GoPlan-Finance/goplan-app#51] [Ideas]
We could us a a simple indicator to show how diverse your portfolio is. Range between 0 and 100%.

Calculation could be based on asset classes (stocks, bonds, cash etc.), sectors, regions, correlation and market cap represented in your portfolio.

https://www.investopedia.com/terms/d/diversification.asp
https://quant.stackexchange.com/questions/10128/how-to-score-a-portfolios-diversity-based-on-security-returns
https://en.wikipedia.org/wiki/Modern_portfolio_theory#Diversification
https://seekingalpha.com/article/1312601-how-to-assess-real-world-portfolio-diversification

Example:
<img src="https://user-images.githubusercontent.com/35574021/110593523-f8cbd300-817b-11eb-942a-2dfc560f7fa9.jpeg" width=400>

**Comment by sadortun** [comment]
Good idea, I think it's also closely related to the concept of [Risk Exposure](https://www.investopedia.com/terms/m/marketexposure.asp) where we should have some red flag when a user have too much assets in one sector.

**Comment by sadortun** [comment]
I would also see a page where you have a list of "health checks" that guide the user with a list of his good/risky behaviours.

- 34% of your portfolio is in high volatile stocks
- You only invested in NASDAQ, you may want to also trade on other exchanges
- .....

**Reply by mkitzmann** [reply]
Love the idea of `Health Checks`! It could make the whole topic of risk assesment more approachable and also lead to clear actions the user should take to avoid these risks. I will open up another discussion concerning this.
